From 375c137b919009ea9934adf3c2e624591bd2bc52 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Tue, 13 Feb 2018 10:28:02 +0800
Subject: [PATCH] drivers/regulator/syr82x: fix compile warning

drivers/regulator/syr82x.c:451:3: warning: this if
clause does not guard... [-Wmisleading-indentation]

Change-Id: Ibdb1f6f903d1e0dece7b58aa3ec89d075e6d5b79
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/regulator/syr82x.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/drivers/regulator/syr82x.c b/drivers/regulator/syr82x.c
index 40af7e507a2e..4f7b0892a3bc 100755
--- a/drivers/regulator/syr82x.c
+++ b/drivers/regulator/syr82x.c
@@ -450,12 +450,12 @@ static int syr82x_i2c_probe(struct i2c_client *i2c, const struct i2c_device_id *
 		config.driver_data = syr82x;
 		if (syr82x->dev->of_node)
 			config.of_node = pdev->of_node[0];
-			if (reg_data && reg_data->constraints.name)
-				rail_name = reg_data->constraints.name;
-			else
-				rail_name = regulators[0].name;
-			reg_data->supply_regulator = rail_name;
-	
+		if (reg_data && reg_data->constraints.name)
+			rail_name = reg_data->constraints.name;
+		else
+			rail_name = regulators[0].name;
+		reg_data->supply_regulator = rail_name;
+
 		config.init_data =reg_data;
 		sy_rdev = regulator_register(&regulators[0],&config);
 		if (IS_ERR(sy_rdev)) {
-- 
2.35.3

